config/datocms/migrations/1734959319_customFileSlugs.ts

```diff
@@ -0,0 +1,50 @@
+import type { Client } from '@datocms/cli/lib/cma-client-node';
+
+export default async function (client: Client) {
+  console.log('Creating new fields/fieldsets');
+
+  console.log(
+    'Create Single-line string field "Slug" (`slug`) in model "\uD83D\uDCE6 File" (`file`)'
+  );
+  await client.fields.create('GjWw8t-hTFaYYWyc53FeIg', {
+    id: 'cnnDYybJTAGvHxiYq2MJ8g',
+    label: 'Slug',
+    field_type: 'string',
+    api_key: 'slug',
```

Comment:
I think naming this slug is confusing. In my perception, a *slug* is a human-readable, yet machine-safe path *fragment*. What is described in the hint, and in the `customFields` function seems to show a *path*, not a *slug*